### PR TITLE
Move the owners file to the root of the repository.

### PR DESCRIPTION
**Pull request by roberthbailey**
Fixes https://github.com/googleforgames/agones/issues/1112

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,10 +4,11 @@ approvers:
...
   - dzlier-gcp
   - EricFortin
-  - ilkercelikyilmaz
-  - jkowalski
-  - joeholley
   - markmandel
   - pooneh-m
   - roberthbailey
 
+emeritus_approvers:
```
Comment (pooneh-m) [inline OWNERS]: Does this mean this group cannot approve anymore or will not be suggested as reviewer?

Comment (roberthbailey) [reply OWNERS]: Yes (to both). See https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus.

Comment (pooneh-m) [reply OWNERS]: Thanks. Does it make sense that the same set of people who their role is being changed to emeritus_approvers, approve the change or if not responding then someone else approve? 

The same [link](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus) says:

>  **They** are encouraged to add **themselves** as an "emeritus" approver under the emeritus_approvers key.

**Comment by pooneh-m** [comment]
> Also move Googlers to emeritus_approvers who aren't actively doing code reviews.

Can we separate the two PRs for emeritus_approvers and moving the file to root directory? I am fine with the file move, but prefer the impacted approvers to approve being moved to emeritus_approvers.

**Comment by roberthbailey** [comment]
I removed the second commit and updated the PR text.

**Comment by roberthbailey** [comment]
The second commit is now a separate PR (https://github.com/googleforgames/agones/pull/1116).

**Pull request merged**